src/test/java/com/exasol/adapter/document/files/TestContainer.java

```diff
@@ -0,0 +1,42 @@
+package com.exasol.adapter.document.files;
+
+import com.azure.storage.file.datalake.DataLakeDirectoryClient;
+import com.azure.storage.file.datalake.DataLakeFileClient;
+import com.azure.storage.file.datalake.DataLakeFileSystemClient;
+import com.azure.storage.file.datalake.DataLakeServiceClient;
+import com.azure.storage.file.datalake.models.PathItem;
+import com.exasol.adapter.document.files.adlstestsetup.AdlsTestSetup;
+import lombok.Getter;
+
+public class TestContainer implements AutoCloseable {
+    @Getter
+    private final DataLakeFileSystemClient dataLakeFileSystemClient;
+    @Getter
+    private final String adlsContainerName;
+
+    public TestContainer(final AdlsTestSetup testSetup) {
+        final String containerName = "adls-document-vs-test-" + System.currentTimeMillis();
+        final DataLakeServiceClient adlsServiceClient = testSetup.getAdlsServiceClient();
+        this.dataLakeFileSystemClient = adlsServiceClient.createFileSystem(containerName);
+        this.adlsContainerName = containerName;
+    }
+
+    //https://github.com/Azure/azure-sdk-for-java/issues/10180
+    public void empty() {
```

Comment:
Maybe rename to "deleteContainerContent()"?

Reply:
It's uniform across all VSs so leaving this like that, I think it's plenty clear.